opt/scopeserver/__init__.py

```diff
@@ -69,45 +70,20 @@ def run(self) -> None:
 
 
 def log_ascii_header() -> None:
-    logger.info(
-        """\n
-
-  /$$$$$$   /$$$$$$                                       /$$$$$$
- /$$__  $$ /$$__  $$                                     /$$__  $$
-| $$  \__/| $$  \__/  /$$$$$$   /$$$$$$   /$$$$$$       | $$  \__/  /$$$$$$   /$$$$$$  /$$    /$$ /$$$$$$   /$$$$$$
-|  $$$$$$ | $$       /$$__  $$ /$$__  $$ /$$__  $$      |  $$$$$$  /$$__  $$ /$$__  $$|  $$  /$$//$$__  $$ /$$__  $$
- \____  $$| $$      | $$  \ $$| $$  \ $$| $$$$$$$$       \____  $$| $$$$$$$$| $$  \__/ \  $$/$$/| $$$$$$$$| $$  \__/
- /$$  \ $$| $$    $$| $$  | $$| $$  | $$| $$_____/       /$$  \ $$| $$_____/| $$        \  $$$/ | $$_____/| $$
-|  $$$$$$/|  $$$$$$/|  $$$$$$/| $$$$$$$/|  $$$$$$$      |  $$$$$$/|  $$$$$$$| $$         \  $/  |  $$$$$$$| $$
- \______/  \______/  \______/ | $$____/  \_______/       \______/  \_______/|__/          \_/    \_______/|__/
-                              | $$
-                              | $$
-                              |__/
-        """
-    )
+    with open(Path("data") / Path("motd.txt")) as motd:
```

Comment:
```suggestion
    with open(Path(__file__).parents[1] / Path("data") / Path("motd.txt")) as motd:
```

This only works when running from opt, which is a bit annoying. Resolving the path relative to the init script will fix this.

Reply:
I've added a config item called `"data"` to find this file (defaults to "data/")

Reply:
For the `"data"` config option, you should add this to the highest level `config.json`. However, this will still fail to run on a new install which hasn't been configured or when running outside of the `opt` dir, it is such an irrelevant bit to fail on. 

Can we add try/except for the file actually existing? Either falling back to not showing the MOTD, or resolving the relative path like in the suggestion above?

Reply:
Ah good point. I've added an `is_file()` existence check with a fallback to, "Welcome to SCope." if it doesn't exist.

Reply:
Thanks. Last things is to add something to document the entry in `config.json` in the root of the repo. A default is good or also an empty key would work.

Reply:
Done. Thanks